ietf-scitt-charter.md

```diff
@@ -1,51 +1,57 @@
-methodologies) is no longer adequate.
+Over the years rapid technological advancements have motivated organizations to implement efficient processes for supply chains (i.e., from design to manufacturing, logistics and just-in time delivery).
+While these improvements help organizations increase efficiency and swiftly bring innovations to market, the rapid increase in scale, size and complexity of supply chain have led to more and more sophisticated supply chain attacks.
+The traditional methods of safeguarding supply chains (e.g. pre and post audit methodologies) is no longer adequate.
 
 Problem Statement
 =================
-Here are the fundamental security issues that face the supply chain ecosystem today:
+It is difficult to manage the compliance of products across end-to-end, global supply chains. 
+
+Some of the fundamental security issues that face the supply chain ecosystem today are as follows:
+
 1. A single product is composed of multiple sub-products coming from different suppliers. There is no agreed upon standard to compose information from different producers
 2. No APIs defined for automated publishing, retrieval or independent verification of the information above.
 3. Absence of decentralized globally inter-operable transparency services to publish the supply chain data in a transparent manner
 4. Insufficient standards for independently verifying the presence of supply chain data in tamper proof data stores 
 
-
-A minimally, simple and concise set of building blocks that interact in a standardized way could guarantee a long term accountability and inter-operability for supply chain components throught their lifecylces across architecturally diverse systems.
+A minimal, simple and concise set of building blocks that interact in a standardized way will serve to assure long term accountability and inter-operability for supply chain components throughout their lifecycles across architecturally diverse systems.
 
 Goals
 =====
-The WG has defined a proposed architecture (link to architecture) for securing a software supply chain. The WG will standardize the overall workflow and essential building blocks that make up the architecture.
+Based on an input document on the architecture (draft-birkholz-scitt-architecture-00), the WG will: 
 
-The WG will use the existing work already done within other IETF WGs such as 
-COSE WG, IETF RATS WG, as appropriate.
+1. standardize the overall security flows for securing a software supply chain, covering the essential building blocks that make up the architecture, and
+2. specify these building blocks, employing the existing work already done within other IETF WGs such as COSE WG, IETF RATS WG, as appropriate.
```

Comment:
I would like to coordinate with W3C WGs as well, specifically the DID WG and the VC WG.